taxi/models.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63,
+                            unique=True)
+    country = models.CharField(max_length=63)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255,
+                                      unique=True)
+
+    class Meta:
+        ordering = ["username"]
+        verbose_name = "Driver"
+        verbose_name_plural = "Drivers"
+
+    def __str__(self):
+        return f"{self.username} {self.first_name} {self.last_name}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer,
+                                     on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,

```

Comment:
use Driver here

Reply:
db.sqlite3 in .gitgnore file. I don't understand, how remove db from commit